src/scss/_layout.scss

```diff
@@ -36,6 +37,17 @@
     padding-right: ($ph-gutter / 2);
     padding-left: ($ph-gutter / 2);
   }
+
+  &.clean {
```

Comment:
Do you think we should rename `.clean` in something more specific? `.clean`. Something like `.ph-item-full`, `.ph-item-border-0`, `.ph-item-*`.